lib/Parse/ParseDecl.cpp

```diff
@@ -3945,7 +3945,10 @@ VarDecl *Parser::parseDeclVarGetSet(Pattern *pattern,
   if (!TyLoc.hasLocation()) {
     if (accessors.Get || accessors.Set || accessors.Addressor ||
         accessors.MutableAddressor) {
-      diagnose(pattern->getLoc(), diag::computed_property_missing_type);
+      SourceLoc locAfterPattern = pattern->getLoc().getAdvancedLoc(
```

Comment:
That entry point is generally used by code completion, but it could work in very limited situations (e.g., getting is just a `return` statement).